torch_geometric/utils/spmm.py

```diff
@@ -31,18 +31,20 @@ def spmm(src: Adj, other: Tensor, reduce: str = "sum") -> Tensor:
 
     :rtype: :class:`Tensor`
     """
-    assert reduce in ['sum', 'add', 'mean', 'min', 'max']
+    assert reduce in [
+        'sum', 'add', 'mean', 'min', 'max'
+    ], (reduce, " reduction is not supported for `torch.sparse.Tensor`.")
 
-    if isinstance(src, SparseTensor):
-        return torch_sparse.matmul(src, other, reduce)
-
-    if not is_torch_sparse_tensor(src):
-        raise ValueError("`src` must be a `torch_sparse.SparseTensor` "
-                         f"or a `torch.sparse.Tensor` (got {type(src)}).")
+    reduce = 'sum' if reduce == 'add' else reduce
 
-    if reduce in ['sum', 'add']:
-        return torch.sparse.mm(src, other)
-
-    # TODO: Support `mean` reduction for PyTorch SparseTensor
-    raise ValueError(f"`{reduce}` reduction is not supported for "
-                     f"`torch.sparse.Tensor`.")
+    if isinstance(src, SparseTensor):
```

Comment:
Both `torch_geometric/utils/__init__.py` and here need to check torch version.

Reply:
ok